AppGenesis/APIStore.swift

```diff
@@ -0,0 +1,57 @@
+
+
+import Foundation
+
+class APIStore: ObservableObject {
+    @Published var apiData: APIModel?
+    
+    init() {
+        loadAPIData()
+    }
+    
+    private func loadAPIData() {
+        if let fileURL = Bundle.main.url(forResource: "apilist", withExtension: "json") {
+            do {
+                let data = try Data(contentsOf: fileURL)
+                let decoder = JSONDecoder()
+                apiData = try decoder.decode(APIModel.self, from: data)
+            } catch {
+                print("Error loading data from app bundle: \(error.localizedDescription)")
+            }
+        } else {
+            print("API data file not found in app bundle.")
+            if let documentDirectoryURL = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first {
```

Comment:
We can also do by extending FileManager and adding documentDirectoryURL, please check that solution.